src/test/scala/utils/TutorialRunner.scala

```diff
@@ -6,37 +6,23 @@ import scala.collection.mutable.ArrayBuffer
 import scala.util.Properties.envOrElse
 
 object TutorialRunner {
-  def apply(tutorialMap: Map[String, String => Boolean], args: Array[String]): Unit = {
-    // Choose the default backend based on what is available.
-    lazy val firrtlTerpBackendAvailable: Boolean = {
-      try {
-        val cls = Class.forName("chisel3.iotesters.FirrtlTerpBackend")
-        cls != null
-      } catch {
-        case e: Throwable => false
-      }
-    }
-    lazy val defaultBackend = if (firrtlTerpBackendAvailable) {
-      "firrtl"
-    } else {
-      ""
-    }
-    val backendName = envOrElse("TESTER_BACKENDS", defaultBackend).split(" ").head
+  def apply(tutorialMap: Map[String, Array[String] => Boolean], args: Array[String]): Unit = {
     val problemsToRun = if(args.isEmpty || args.head == "all" ) {
```

Comment:
I don't think this is going to work, if there are flags followed by "all" this will not get the "all"